src/components/IOUConfirmationList.js

```diff
@@ -0,0 +1,248 @@
+import Text from './Text';
+import themeColors from '../styles/themes/default';
+import {
+    getIOUConfirmationOptionsFromMyPersonalDetail,
+    getIOUConfirmationOptionsFromParticipants,
+} from '../libs/OptionsListUtils';
+import OptionsList from './OptionsList';
+import ButtonWithLoader from './ButtonWithLoader';
+import ONYXKEYS from '../ONYXKEYS';
+
+const propTypes = {
+    // Callback to inform parent modal of success
+    onConfirm: PropTypes.func.isRequired,
+
+    // callback to update comment from IOUModal
+    onUpdateComment: PropTypes.func,
+
+    // comment value from IOUModal
+    comment: PropTypes.string,
+
+    // Should we request a single or multiple participant selection from user
+    hasMultipleParticipants: PropTypes.bool.isRequired,
+
+    // IOU amount
+    iouAmount: PropTypes.string.isRequired,
+
+    // Selected currency from the user
+    // remove eslint disable after currency symbol is available
+    // eslint-disable-next-line react/no-unused-prop-types
+    selectedCurrency: PropTypes.string.isRequired,
+
+    // Selected participants from IOUMOdal with login
+    participants: PropTypes.arrayOf(PropTypes.shape({
+        login: PropTypes.string.isRequired,
+        alternateText: PropTypes.string,
+        hasDraftComment: PropTypes.bool,
+        icons: PropTypes.arrayOf(PropTypes.string),
+        searchText: PropTypes.string,
+        text: PropTypes.string,
+        keyForList: PropTypes.string,
+        isPinned: PropTypes.bool,
+        isUnread: PropTypes.bool,
+        reportID: PropTypes.number,
+        participantsList: PropTypes.arrayOf(PropTypes.object),
+    })).isRequired,
+
+    /* Onyx Props */
+
+    // The personal details of the person who is logged in
+    myPersonalDetails: PropTypes.shape({
+
+        // Display name of the current user from their personal details
+        displayName: PropTypes.string,
+
+        // Avatar URL of the current user from their personal details
+        avatar: PropTypes.string,
+
+        // Primary login of the user
+        login: PropTypes.string,
+    }).isRequired,
+
+    // Holds data related to IOU view state, rather than the underlying IOU data.
+    iou: PropTypes.shape({
+
+        // Whether or not the IOU step is loading (creating the IOU Report)
+        loading: PropTypes.bool,
+    }),
+};
+
+const defaultProps = {
+    iou: {},
+    onUpdateComment: null,
+    comment: '',
+};
+
+class IOUConfirmationList extends Component {
+    /**
+     * Returns the sections needed for the OptionsSelector
+     *
+     * @param {Boolean} maxParticipantsReached
+     * @returns {Array}
+     */
+    getSections() {
+        const sections = [];
+
+        if (this.props.hasMultipleParticipants) {
+            const formattedMyPersonalDetails = getIOUConfirmationOptionsFromMyPersonalDetail(
+                this.props.myPersonalDetails,
+
+                // convert from cent to bigger form
+                `$${this.calculateAmount(true) / 100}`,
```

Comment:
Same here, can you mention that USD is only temporary and that all currencies need to be supported